Description: Make plainbox.impl.secure.launcher1 executable via `python -m`
 This patch adds a simple if __name__ == '__main__': main() statement
 a the end of plainbox.impl.secure.launcher1 module so that it can be
 executed directly with `python3 -m ...`. This is useful to generate
 the plainbox-trusted-launcher-1.1 manual page with help2man

Author: Zygmunt Krynicki <zygmunt.krynicki@canonical.com>
Bug: https://bugs.launchpad.net/checkbox/+bug/1255085
Last-Update: 2013-11-26
Forwarded: yes

--- plainbox-0.4~b1.orig/plainbox/impl/secure/launcher1.py
+++ plainbox-0.4~b1/plainbox/impl/secure/launcher1.py
@@ -173,3 +173,7 @@ def main(argv=None):
             return launcher.run_shell_from_job(ns.hash, env)
         except LookupError as exc:
             raise SystemExit(str(exc))
+
+
+if __name__ == "__main__":
+    main()
